jsapp/js/account/organization/MemberActionsDropdown.tsx

```diff
@@ -0,0 +1,104 @@
+// Libraries
+import {useState} from 'react';
+import cx from 'classnames';
+
+// Partial components
+import KoboDropdown from 'jsapp/js/components/common/koboDropdown';
+import Button from 'jsapp/js/components/common/button';
+import MemberRemoveModal, {REMOVE_SELF_TEXT} from './MemberRemoveModal';
+
+// Stores, hooks and utilities
+import {useSession} from 'jsapp/js/stores/useSession';
+import {getSimpleMMOLabel} from './organization.utils';
+import envStore from 'jsapp/js/envStore';
+import subscriptionStore from 'jsapp/js/account/subscriptionStore';
+
+// Constants and types
+import {OrganizationUserRole} from './organizationQuery';
+
+// Styles
+import styles from './memberActionsDropdown.module.scss';
+
+interface MemberActionsDropdownProps {
+  /** Target member username. */
+  username: string;
```

Comment:
Might be nice to rename this to something like `targetUsername` and then we won't need the comment.